languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -30195,7 +30196,7 @@ USA
                 </token>
                 <token postag="SENT_END"/>
             </pattern>
-            <message>Did you mean <suggestion>\2 my</suggestion>, <suggestion>\2 her</suggestion>, <suggestion>\2 his</suggestion>, <suggestion>\2 our</suggestion>, or <suggestion>\2 their</suggestion>?</message>
+            <message>The possessive form of the pronoun may be required here. Did you mean <suggestion>\2 my</suggestion>, <suggestion>\2 her</suggestion>, <suggestion>\2 his</suggestion>, <suggestion>\2 our</suggestion>, or <suggestion>\2 their</suggestion>?</message>
```

Comment:
In this message (as well as the following one), your text would suffice. The suggestions could be taken out of the message.